CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## v1.14.0
+
+<a name="breaking_changes_1.14.0">[Breaking Changes:](#breaking_changes_1.14.0)</a>
+- [terminal] `ShellProcess` does not longer merge the received `ShellProcessOptions` with the process environment variables.  
```

Comment:
```suggestion
- [terminal] `ShellProcess` no longer merges the received `ShellProcessOptions` with the process environment variables.  
```

Reply:
After Paul's comments, I removed the breaking changes, see: 
https://github.com/eclipse-theia/theia/pull/9437#discussion_r627587900